cypress/PageObjects/addNewElement.js

```diff
@@ -0,0 +1,30 @@
+///reference types="Cypress"
+
+class addNewElement {
```

Comment:
AddNewElementPage (name first letter in uppercase, *Page in the end of the name)